### PR TITLE
Add sensei preview course

### PR DESCRIPTION
**Pull request by rawrrencee**
will update once obj is updated on backend side to return firstNme/lastName/profileImgUrl
addresses #118 

from telegram: _can i request for Reviews of a course to also join the User table to return basic details (firstName, lastName, profileImgUrl) of the user that wrote the review?_

# Changelog:

Add Sensei view Course as Student functionality
Add all reviews modal to show all reviews of the course (to be updated when review object returns the basic user details about who posted the review)
Remove the comments tab from the Create Lesson modal.
Align profile pic and Reviews box.

## Checklist:

- [ ] Merged latest develop
- [ ] PR title makes sense

## Screenshots (where relevant):
image
image
image

### CODE REVIEW

File: src/pages/student/dashboard/courses/view/index.js [hunk 1]
```diff
@@ -120,17 +124,30 @@ const StudentCourseDetails = () => {
...
+            {user.userType !== USER_TYPE_ENUM.STUDENT && (
+              <Button
+                type="default"
+                size="large"
+                shape="round"
+                onClick={() => setShowAllReviewsModal(true)}
+                icon={<FormOutlined />}
+              >
+                Reviews
+              </Button>
```
Comment (natalieloke) [inline src/pages/student/dashboard/courses/view/index.js]: other students cannot see reviews of the same course? 

also wondering if the FormOutlined icon would indicate the idea of filling in a form but we are just clicking on it to view reviews

**Review by natalieloke — COMMENTED** [review]
in terms of UX, when I think in terms of a user (student/admin), i would probably prefer to view all reviews at one shot when I am viewing the course! means like it's not nested in the modal in the button and it's in the same level as the course description etc. 

@rawrrencee  what do you think?

File: src/pages/student/dashboard/courses/view/index.js (hunk 1 above)
Comment (rawrrencee) [reply src/pages/student/dashboard/courses/view/index.js]: oh yeah! okay then i'll just remove the reviews portion and add it in again later as a separate Reviews card!

**Comment by rawrrencee** [comment]
Hey @natalieloke I removed the Reviews button + AllReviewsModal from the View Course page. Will add in a AllReviews card component in the main page instead, which will have pagination later on when backend introduces the User data to each Review object!

File: src/pages/student/dashboard/courses/view/index.js [hunk 2]
```diff
@@ -120,17 +122,19 @@ const StudentCourseDetails = () => {
...
+                size="large"
+                shape="round"
+                onClick={() => {
+                  setShowReviewModal(true)
+                }}
+                icon={<EditOutlined />}
+              >
+                {`${editMode ? 'Edit your' : 'Add a'}  Review`}
+              </Button>
+            )}
```
Comment (rawrrencee) [inline src/pages/student/dashboard/courses/view/index.js:137]: Left the userType check here since Admin/Sensei cannot review a course

Comment (rawrrencee) [reply src/pages/student/dashboard/courses/view/index.js:137]: @natalieloke

Comment (natalieloke) [reply src/pages/student/dashboard/courses/view/index.js:137]: Yep I saw this in the earlier review and this is fine

**Comment by natalieloke** [comment]
> Hey @natalieloke I removed the Reviews button + AllReviewsModal from the View Course page. Will add in a AllReviews card component in the main page instead, which will have pagination later on when backend introduces the User data to each Review object!

Thanks, will the AllReviews card component be coming in a separate PR? @rawrrencee

**Comment by rawrrencee** [comment]
> > Hey @natalieloke I removed the Reviews button + AllReviewsModal from the View Course page. Will add in a AllReviews card component in the main page instead, which will have pagination later on when backend introduces the User data to each Review object!
> 
> Thanks, will the AllReviews card component be coming in a separate PR? @rawrrencee

Yup, won't be in this one! This one just for making the Sensei be able to Preview Course like how a student will see it, and respond to comments inside. (which was sth raised by prof in the 2nd sys rel presentation)

**Review by natalieloke — APPROVED** [review]

**Pull request merged**